src/components/TopBar/Wallet/Assets/index.tsx

```diff
@@ -230,6 +242,34 @@ const AssetsIndex: FC<OHMAssetsProps> = (props: { path?: string }) => {
               );
           }
         })()}
+        {activeChain.id === 5 && (
```

Comment:
Extract `5` into a constant and put it into the constants file. It'll be referred to elsewhere, too.

Reply:
yes, i agree. its already extracted in the networkDetails file. should import networkDetails and use NetworkId.TESTNET_GOERLI

Reply:
done, don't know why I didn't user NetworkId.TESTNET_GOERLI in the first place